artifacts/LogonScript.ps1

```diff
@@ -19,611 +19,73 @@ $DisableArcAgentAutoUpgrade = if ($env:disableArcAgentAutoUpgrade) { $env:disabl
-    exit -1
-}
-
-# Ensure `connectedk8s` az cli extension is installed and up to date.
-$errOut = $($retVal = & {az extension add --upgrade --name connectedk8s -y --allow-preview false}) 2>&1
-if ($LASTEXITCODE -ne 0)
-{
-    throw "Error upgrading extension connecktedk8s : $errOut"
-}
-
-$installDir = $((Get-Location).Path)
-$productName = "AKS Edge Essentials - K3s"
-$networkplugin = "flannel"
-if ($UseK8s) {
-    $productName ="AKS Edge Essentials - K8s"
-    $networkplugin = "calico"
-    # Setting URL to empty string, so K8s msi will be selected
-    $productUrl = ""
-}
-
-# Here string for the json content
-$aideuserConfig = @"
-{
-    "SchemaVersion": "1.1",
-    "Version": "1.0",
-    "AksEdgeProduct": "$productName",
-    "AksEdgeProductUrl": "$productUrl",
-    "Azure": {
-        "SubscriptionName": "",
-        "SubscriptionId": "$SubscriptionId",
-        "TenantId": "$TenantId",
-        "ResourceGroupName": "$ResourceGroupName",
-        "ServicePrincipalName": "aksedge-sp",
-        "Location": "$Location",
-        "CustomLocationOID":"$CustomLocationOid",
-        "EnableWorkloadIdentity": true,
-        "EnableKeyManagement": true,
-        "GatewayResourceId": "",
-        "Auth":{
-            "ServicePrincipalId":"",
-            "Password":""
-        }
-    },
-    "AksEdgeConfigFile": "aksedge-config.json"
-}
-"@
-$aksedgeConfig = @"
-{
-    "SchemaVersion": "1.14",
-    "Version": "1.0",
-    "DeploymentType": "SingleMachineCluster",
-    "Init": {
-        "ServiceIPRangeSize": 10
-    },
-    "Network": {
-        "NetworkPlugin": "$networkplugin",
-        "InternetDisabled": false,
-        "Proxy": {
-            "Http": null,
-            "Https": null,
-            "No": null
-        }
-    },
-    "User": {
-        "AcceptEula": true,
-        "AcceptOptionalTelemetry": true
-    },
-    "Machines": [
-        {
-            "LinuxNode": {
-                "CpuCount": "$CpuCoreCount",
-                "MemoryInMB": $VMMemory,
-                "DataSizeInGB": 40,
-                "LogSizeInGB": 4
-            }
-        }
-    ]
-}
-"@
+# download public script and config json tempaltes
+$scriptUrl = "https://raw.githubusercontent.com/jagadishmurugan/AKS-Edge/refs/heads/users/jagamu/pvt-accept-input-from-config-file/tools/scripts/AksEdgeQuickStart/AksEdgeQuickStartForAio.ps1"
+$scriptPath = "AksEdgeQuickStartForAio.ps1"
 
-###
-# Main
-###
-if (-not (Test-Path -Path $installDir)) {
-    Write-Host "Creating $installDir..."
-    New-Item -Path "$installDir" -ItemType Directory | Out-Null
-}
+Invoke-WebRequest -Uri $scriptUrl -OutFile $scriptPath
 
-$starttime = Get-Date
-$starttimeString = $($starttime.ToString("yyMMdd-HHmm"))
-$transcriptFile = "$installDir\aksedgedlog-$starttimeString.txt"
+# download the aio-aide-userconfig.json file
+$userConfigUrl = "https://raw.githubusercontent.com/Azure/AKS-Edge/d03e9228dd7a0fc4c291bf792bebf81fcc7f4bba/tools/aio-aide-userconfig.json"
```

Comment:
This is a specific version, right? 
We want to get the latest?

Reply:
updated to main